src/App.tsx

```diff
@@ -1,203 +1,88 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import classNames from 'classnames';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState(Status.all);
+  const [isEditingTodo, setIsEditingTodo] = useState<Todo | null>(null);
```

Comment:
Better use 'is' in the variable name when it means boolean. You can change it to the name from Readme.md - tempTodo.